LICENSE.txt

```diff
@@ -1 +1,54 @@
-DS EULA License
\ No newline at end of file
+StreamX End-User License Agreement
+Version 1.0, April 2024
+https://www.streamx.dev/licenses/eula-v1-0.html
+
+This End User License Agreement (“EULA”) is a legal agreement between you and Dynamic Solutions sp. z o.o. sp.k., a limited partnership created under the law of Poland with its registered office in Białystok, address: ul. Poleska 45/1, 15-114 Białystok, entered into the entrepreneurs register under KRS no 0000573359 (“DS”). It covers the terms and conditions for using StreamX in its executable form and any updates that may be made available if or when available (“Software”).
+
+IF YOU DO NOT AGREE TO ALL OF THE TERMS OF THIS EULA, DO NOT INSTALL, USE, OR COPY THE SOFTWARE.
+
+General
+
+- StreamX is a real-time digital experience mesh. It streamlines the integration of content management systems with other backend systems needed to create digital experiences.
+- You must agree to all of the terms of this EULA to use this Software.
+- This Software may automatically communicate with DS servers to receive and install updates, and to send anonymized usage information. You agree to allow this communication.
+- This Software is provided “as-is” with no warranties, and you agree that DS is not liable for anything you do with it.
+
+Agreement
+
+By downloading, installing, using, or copying the Software, you accept and agree to be bound by the terms of this EULA. If you do not agree to all of the terms of this EULA, you may not download, install and use the Software.
+
+License
+
+The Licensor hereby grants to the Licensee a non-exclusive, free of charge, limited, and revocable license to use the Software for trial and internal educational purposes only.
+
+Restrictions
+
+Under this EULA you may not:
+- reverse-engineer, decompile, disassemble, or attempt to derive the source code for the Software, in whole or in part.
+- license, sublicense, sell, resell, rent, lease, or exchange the Software.
+- make amendments to the source code of the Software
+- use the Software for any external, including commercial, purposes including placing it or products integrated with it on the market,
+- make the Software available to third parties, copy the Software for publication or distribution,
+- use the Software to create separate works, except for copies of the works created for trial purposes.
+
+This EULA can be revoked by DS at any time. In such case you are obliged to immediately cease the use of the Software and permanently remove all its components from your infrastructure and all data storage devices.
+
+Privacy
+
+The Software will not communicate with DS servers therefore no personal data will be processed by us.
```

Comment:
In website there is such text in this section:
```
The Software may communicate with DS servers only to receive and install updates, and to send anonymized usage information including (IP) address, and the date and time the Software was used, therefore no personal data will be processed by us.```

Reply:
changed